versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.3.0"
```

Comment:
Please don't increase the terraform this far. We have many users still using the old version.

I believe we also ran into an issue with optional inputs that @Nuru ran into.

Best of both worlds is to use a type = any imho to make all the inputs optional.